### PR TITLE
[CDAP-20982] Removing Hbase code references as a part of Hadoop upgrade 2 to 3 

### PR DESCRIPTION
**Pull request by sahusanket**

### CODE REVIEW

**Comment by sahusanket** [comment]
Hi @chtyim ,
Please review this PR for removing hbase

- I have tried to replace hbase with levelDB for some implementations and injections. 
- Removed hbase references in JAVA code. 
- There are few parts i am not sure : i.e. there are parts were only hbase is being used. 

   - `cdap-common/bin/functions.sh`
   - `cdap-master/src/main/java/io/cdap/cdap/data/runtime/main/MasterServiceMain.java` 
   - I have removed hbase refs but i am not sure if we still support the above usecases ? like cdap scripts ? 
   
 Testing : 
- Build sandbox, and ran basic operations, connections, pipeline run etx..all successful 
- Ran cdap-ui e2e test on above sandbox, seems successful 

- Replaced image in a GKE  cluster ( CDF ) and tested basic operations/runs and it worked successfully. 

Please let me know are there any other testing that i need to perform to ensure everything is working fine.

File: cdap-common/src/main/java/io/cdap/cdap/common/messaging/MessagingUtils.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package io.cdap.cdap.messaging;
+package io.cdap.cdap.common.messaging;
```
Comment (tivv) [inline cdap-common/src/main/java/io/cdap/cdap/common/messaging/MessagingUtils.java]: It looks like this class is used in cdap-tms and cdap-watchdog. cdap-watchdog has tms as dependency. Can we move it to cdap-tms and leave in the original package?

**Review by tivv — COMMENTED** [review]
I see that cdap-data-fabric-tests failed. Could you check it?

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/batch/BasicMapReduceTaskContext.java [hunk 2]
```diff
@@ -383,11 +382,12 @@ public <T extends Dataset> T getDataset(String namespace, String name,
    * true.
    */
   private Map<String, String> adjustRuntimeArguments(Map<String, String> args) {
-    if (args.containsKey(HBaseTable.SAFE_INCREMENTS)) {
+    String SAFE_INCREMENTS = "dataset.table.safe.readless.increments";
```
Comment (tivv) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/batch/BasicMapReduceTaskContext.java]: I think this was use only in HBaseTable, we can remove it altogether

File: cdap-data-fabric-tests/src/test/java/io/cdap/cdap/data2/transaction/distributed/TransactionServiceClientTest.java [hunk 3]
```diff
@@ -90,9 +91,10 @@ protected TransactionStateStorage getStateStorage() {
 
   @BeforeClass
   public static void beforeClass() throws Exception {
-    HBaseTestingUtility hBaseTestingUtility = new HBaseTestingUtility();
-    hBaseTestingUtility.startMiniDFSCluster(1);
-    Configuration hConf = hBaseTestingUtility.getConfiguration();
+    Configuration hConf = new Configuration();
+    hConf.set(MiniDFSCluster.HDFS_MINIDFS_BASEDIR,tmpFolder.newFolder().getAbsolutePath());
```
Comment (tivv) [inline cdap-data-fabric-tests/src/test/java/io/cdap/cdap/data2/transaction/distributed/TransactionServiceClientTest.java]: Please fix checkstyle

File: cdap-data-fabric-tests/src/test/java/io/cdap/cdap/data2/transaction/distributed/TransactionServiceTest.java [hunk 4]
```diff
@@ -50,10 +50,12 @@
 import io.cdap.cdap.spi.metadata.noop.NoopMetadataStorage;
 import java.io.File;
 import java.util.Arrays;
+import java.util.Map;
```
Comment (tivv) [inline cdap-data-fabric-tests/src/test/java/io/cdap/cdap/data2/transaction/distributed/TransactionServiceTest.java]: Please fix imports

Comment (tivv) [reply cdap-data-fabric-tests/src/test/java/io/cdap/cdap/data2/transaction/distributed/TransactionServiceTest.java]: Also please fix other checkstyle inssues in new code

File: pom.xml [hunk 5]
```diff
@@ -2039,7 +1828,8 @@
         <package.rpm.arch>all</package.rpm.arch>
         <package.dirs>opt etc</package.dirs>
         <package.config.dirs>--config-files etc/cdap --config-files etc/logrotate.d --config-files etc/security</package.config.dirs>
-        <dependency.groupId.exclusions>org.apache.hadoop,org.apache.hbase,\
+        <dependency.groupId.exclusions>org.apache.hadoop,\
+<!--          org.apache.hbase,\-->
```
Comment (tivv) [inline pom.xml]: Don't add commented code

**Review by tivv — COMMENTED** [review]
Also please update PR title to include JIRA ID and remove draft reference

**Review by tivv — APPROVED** [review]

**Pull request merged**